modules/react/common/lib/styles/focusRing.ts

```diff
@@ -62,8 +60,6 @@ function calculateFocusRing({
   return {boxShadow};
 }
 
-export const memoizedFocusRing = memoize(calculateFocusRing, (...args) => JSON.stringify(args));
```

Comment:
@NicholasBoll Gone as you said you haven't seen a need for a stable ref

Reply:
There also isn't much to calculate, so the cost of the memoization might be more than running the function.